WordPress/src/main/res/layout/new_quick_start_task_type_item.xml

```diff
@@ -0,0 +1,69 @@
+    android:id="@+id/quick_start_item_root"
+    style="@style/QuickStartTypeRow"
+    android:padding="@dimen/my_site_card_row_padding">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/quick_start_item_title"
+        style="@style/NewQuickStartTypeTitle"
+        android:layout_width="0dp"
+        android:text="@string/quick_start_sites_type_get_to_know_app"
+        app:layout_constraintEnd_toStartOf="@+id/quick_start_item_icon"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/quick_start_item_icon"
+        style="@style/NewQuickStartIcon"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:contentDescription="@null"
+        android:src="@drawable/ic_tour_illustration"
+        app:layout_constraintBottom_toBottomOf="@+id/quick_start_item_progress"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/quick_start_item_title" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/quick_start_item_subtitle"
+        style="@style/QuickStartTypeSubtitle"
+        android:layout_width="0dp"
+        android:layout_marginTop="4dp"
+        app:layout_constraintEnd_toStartOf="@+id/task_completed_icon"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/quick_start_item_title"
+        tools:text="1 of 6 complete" />
+
+    <ImageView
+        android:id="@+id/task_completed_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@null"
+        android:visibility="gone"
+        tools:visibility="visible"
+        android:src="@drawable/ic_baseline_check_24"
+        app:tint="@color/green_40"
+        app:layout_constraintBottom_toBottomOf="@+id/quick_start_item_subtitle"
+        app:layout_constraintEnd_toStartOf="@+id/quick_start_item_icon"
+        app:layout_constraintHorizontal_bias="1.0"
+        app:layout_constraintTop_toTopOf="@+id/quick_start_item_subtitle" />
+
+    <ProgressBar
+        android:id="@+id/quick_start_item_progress"
+        style="?android:progressBarStyleHorizontal"
+        android:layout_width="0dp"
+        android:layout_height="4dp"
+        android:layout_marginTop="@dimen/margin_medium"
+        android:indeterminate="false"
+        android:progressDrawable="@drawable/quick_start_card_horizontal_progress_drawable"
+        app:layout_constraintEnd_toStartOf="@+id/quick_start_item_icon"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/quick_start_item_subtitle"
+        tools:progress="100"
+        tools:progressBackgroundTint="@color/primary_0"
+        tools:progressTint="@color/colorPrimary" />
```

Comment:
Np: This part can be formatted.

Reply:
@ashiagr I have done the formatting in the commit. https://github.com/wordpress-mobile/WordPress-Android/pull/16629/commits/9782625feca58e29b926820934e929a33d627cbd.